roles/openshift_cfme/defaults/main.yml

```diff
@@ -1,42 +1,184 @@
 ---
```

Comment:
The data in this file along with the descriptions feels like it should be in the README.md for the role.  I like how descriptive the text is and maybe this is ok but I haven't seen roles put all their descriptions and explanations inside of their `defaults/main.yml`.

Makes searching for default variables a bit more difficult.

Thoughts?

Reply:
I think we should also ask ourselves how many of these we actually want people to change. If they're constants then move them to vars/main.yml instead. To me these are definitely constants
 - openshift_cfme_project
 - openshift_cfme_service_account
 - Everything you've called out as constants

Reply:
+1 on moving the documentation to README.md over defaults/main.yml

Reply:
👍 as well

Reply:
Yeah. Totes.

Reply:
Lets get this in a follow up ASAP

Reply:
@sdodson would constifying openshift_cfme_project mean i can't easily install multiple versions of cfme on the same cluster (for testing)? this would be unfortunate, as i can see myself wanting to run different versions, and having to have separate clusters for each version sounds like overkill.

Reply:
Yeah it would preclude multiple deployments. I'd like to understand for sure if this is something that we're going to need to support or not. I imagine namespace alone wouldn't be the only thing preventing us from doing multiple deployments into one cluster today.

Reply:
from the way i've deployed cfme (manually) it pretty much just sets everything up inside it's own namespace. not sure how the various pods depend on finding the "right" postgres db though...

i guess i'll find out when i run this role. i already have one cfme setup in the 'cfme' namespace, and this role looks like it uses 'openshift-cfme'.

Reply:
I think the problem will be with storage. Just guessing.

Reply:
Moved the constants over into `vars/`

Reply:
i can now say with confidence that i have two cfme installs on the same cluster. one from my manual install in the 'cfme' namespace, and a new one from this role in the 'openshift-cfme' namespace.